src/shared/components/workbasket-filters/workbasket-filters.component.ts

```diff
@@ -106,8 +112,10 @@ export class WorkbasketFiltersComponent implements OnInit {
   reset(): void {
     this.windowService.removeLocalStorage(FORM_GROUP_VAL_LOC_STORAGE);
     this.windowService.removeLocalStorage(SAVED_QUERY_PARAM_LOC_STORAGE);
-    this.resetFieldsWhenNoDefaults();
-    this.onReset.emit(true);
+    setTimeout (() => {
+      this.resetFieldsWhenNoDefaults();
+      this.onReset.emit(true);
+    }, 1000);
```

Comment:
is the timeout neccessary? if yes can the 1000 reduce to 500 to achieve better performance?

Reply:
i have changed it to 500. I needed to add setTimeout as i found during testing that local storage would sometimes clear just after this.onRese.emit(true) was called

Reply:
ok please test whether 500 is not enough and breaking the time sequence.